lib/utils.js

```diff
@@ -33,6 +34,47 @@ exports.basicAuth = function(username, password) {
 	};
 };
 
+// Matches routes
+exports.matchRoutes = function(req, res) {
+
+  var path = (req.params[0]);
+  res.render(path, function(err, html) {
+    if (err) {
+      res.render(path + "/index", function(err2, html) {
+        if (err2) {
+          console.log(err);
+          res.status(404).send(err + "<br>" + err2);
+        } else {
+          res.end(html);
+        }
+      });
+    } else {
+      res.end(html);
+    }
+  });
+
+};
+
+// Synchronously get the url for the latest release on github
+exports.getLatestRelease = function() {
+  var url = "";
+  var options = {
+    headers: {'user-agent': 'node.js'}
+  }; 
+  var gitHubUrl = 'https://api.github.com/repos/alphagov/govuk_prototype_kit/releases/latest';
+  try {
+      res = request('GET', gitHubUrl, options);
```

Comment:
I guess we should cache the url in some way. Do you think it's enough to just set it once when the app starts, or put some kind of expiry on it?